src/components/Search/SearchResult/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import CardList from '../../CardList';
+
+interface SearchResultProps {
+  items: { name: string; url: string }[];
+  isLoading: boolean;
+  error: string | null;
+}
+
+export default class SearchResult extends React.Component<SearchResultProps> {
+  render() {
+    const { items, isLoading, error } = this.props;
+
+    if (isLoading) {
+      return <p className="padding">Loading...</p>;
+    }
+
+    if (error) {
+      return <p className="error">Error: {error}</p>;
+    }
+
+    if (items.length === 0) {
```

Comment:
Write a helper function that will check if there are elements in the array and return true/false,  this function will be very useful in the future